contracts/NFTMarket.sol

```diff
@@ -51,6 +51,7 @@ contract NFTMarket is
         address seller;
         uint256 price;
         //int128 usdTether; // this would be to "tether" price dynamically to our oracle
+        address targetBuyer;
```

Comment:
Adding this field will almost certainly cause problems wrt upgradeability of the contract. Instead, there should be a new multi-level mapping, similar to `listings` below, exclusively for the `targetBuyer` of a particular listing.

Reply:
That's a good idea, I did not think about it!